docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,101 +1,119 @@
+- Go to **Site admin > Site configuration** (`/site-admin/configuration`) on your instance and set:
 
-The following metrics are available for Cody Enterprise users:
+```json
+    {
+      // [...]
+      "cody.enabled": false
+    }
+```
 
-| **Metric Type** |                                                                                                                                           **What is measured?**                                                                                                                                            |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Active users    | - Total active users <br /> - Average daily users <br /> - Average no. of days each user used Cody (of last 30 days) <br /> - Cody users by day (last 30 days) <br /> - Cody users by month (last two months) <br /> - Cody users by number of days used                                                   |
-| Completions     | - Total accepted completions <br /> - Minutes saved per completion <br /> - Hours saved by completions <br /> - Cody completions by day <br /> - Completions acceptance rate <br /> - Weighted completions acceptance rate <br /> - Average completion latency <br /> - Acceptance rate by language <br /> |
-| Chat            | - Total chat events <br /> - Minutes saved per chat <br /> - Hours saved by chats <br /> - Cody chats by day                                                                                                                                                                                               |
-| Commands        | - Total command events <br /> - Minutes saved per command <br /> - Hours saved by commands <br /> - Cody commands by day <br /> - Most used commands                                                                                                                                                       |
+- Next, remove `completions` configuration if they exist
 
-To enable Cody Analytics:
+## Enable Cody only for some users
 
-- Create an account on [Sourcegraph Accounts](https://accounts.sourcegraph.com/)
-- A user already having a Sourcegraph.com account gets automatically migrated to Sourcegraph Accounts. Users can sign in to Cody Analytics using their email and password
-- Users without a Sourcegraph.com account should contact one of our team members. They can help with both the account setup and assigning instances to specific users
-- Map your user account to a Sourcegraph instance, and this gives you access to Cody analytics
+To enable Cody only for some users it depends on what version of Sourcegraph you are running.
```

Comment:
It's slightly unclear / difficult to grasp what "it" refers to in this sentence.
```suggestion
How to enable Cody only for _some_ users depends on what version of Sourcegraph you are running.
```